engine/src/main/java/org/terasology/rendering/dag/stateChanges/EnableBlending.java

```diff
@@ -29,22 +29,6 @@
 public final class EnableBlending extends EnableStateParameter {
     private static StateChange defaultInstance = new DisableBlending();
 
-    // TODO: Do we really need to add such a description for every single StateChange?
-    // TODO: IMHO, we can just add the common parts to StateChange interface, and add class-specific info as needed.
-    /**
-     * Constructs an instance of this StateChange. This is can be used in a node's initialise() method in
-     * the form:
-     *
-     * addDesiredStateChange(new EnableBlending());
-     *
-     * This trigger the inclusion of an EnableStateParameterTask instance and a DisableStateParameterTask instance
-     * in the rendering task list, each instance enabling/disabling respectively the GL_BLEND mode. The
-     * two task instance frame the execution of a node's process() method unless they are deemed redundant,
-     * i.e. because the upstream or downstream node also enables blending.
-     *
-     * See also StateChange implementation SetBlendFunction to set the source and destination factors
-     * used by the blending process.
-     */
```

Comment:
I see what you mean with this changes, thank you for providing a sample. 

I still think we should have a short constructor javadoc along these lines:

```
/**
Constructor, to be used in the initialise() method of a node:

E.g.   `addDesiredStateChange(new EnableBlending());`
*/
```

In this particular case it's fairly trivial, but there will be state changes with constructor having arguments: in those cases providing an example will be more helpful. 

Anyway, please go ahead with updating/writing javadocs for all the touched or new classes, along these lines.